docs/articles/call-settings.md

```diff
@@ -1,3 +1,37 @@
 # Call settings
 
-(Chris Bacon to write this)
\ No newline at end of file
+## Introduction
+
+All C# [gRPC](http://www.grpc.io/) APIs allow customization of RPC calls by using [CallSettings](../obj/api/Google.Api.Gax.CallSettings.yml). If `CallSettings` are not specified sensible defaults are automatically provided.
```

Comment:
I'd say "all client libraries wrapping gRPC-based APIs" (and feel free to link to api-layers.md). After all, the gRPC API itself uses CallOptions instead, which may be worth mentioning too.